entrypoint.js

```diff
@@ -4,9 +4,23 @@ shim("check-master");
 const {Toolkit} = require('actions-toolkit');
 
 Toolkit.run(async tools => {
-  const path = tools.arguments.path || '.*';
+  let paths = [];
+
+  switch(typeof tools.arguments.path) {
```

Comment:
```suggestion
  if (!Array.isArray(paths)) { 
      paths = [paths];
  }
```

Reply:
Forgot about that issue with arrays, changed it to use `Array.isArray`.